### PR TITLE
filter and idea form styling

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: civic_europe/assets/scss/components/_custom_form.scss [hunk 1]
```diff
@@ -79,6 +79,14 @@
         color: $brand-danger;
         font-size: $font-size-md;
     }
+
+    .help-block {
```
Comment (Rineee) [inline civic_europe/assets/scss/components/_custom_form.scss:83]: How did you know it is this class? And where would I find 'bootstrap3/layout/help_text.html' (which is included in the checkboxselectmultiple.html)? This is all still magic to me :sweat_smile:

**Review by Rineee — APPROVED** [review]
Supercool!!

**Comment by Rineee** [comment]
Haha, but tests are failing bc of organisation -> organization, great!

File: civic_europe/assets/scss/components/_custom_form.scss (hunk 1 above)
Comment (fuzzylogic2000) [reply civic_europe/assets/scss/components/_custom_form.scss:83]: Lot's of inspecting and grepping and trying it out. And I am not sure that really is the way to go, but works. :woman_shrugging:

**Comment by fuzzylogic2000** [comment]
@Rineee the test is fixed, so I will merge to also fix that test on master!

**Pull request merged**